src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -10,6 +10,8 @@ public struct OSPlatform : IEquatable<OSPlatform>
 
         public static OSPlatform Linux { get; } = new OSPlatform("LINUX");
 
+        public static OSPlatform NetBSD { get; } = new OSPlatform("NETBSD");
```

Comment:
The general feeling from the team was we'd like platforms to bake for a while before getting named constants this file (this is consistent with what we are doing with FreeBSD, who's porting effort stalled out) .  For now, can you revert this change and just use  `OSPlatform.Create("NETBSD")` anywhere you need it?